drivers/scheduler/k8s/k8s.go

```diff
@@ -4140,16 +4145,19 @@ func (k *K8s) ResizeVolume(ctx *scheduler.Context, configMapName string) ([]*vol
 		}
 		if obj, ok := specObj.(*corev1.PersistentVolumeClaim); ok {
 			updatedPVC, _ := k8sCore.GetPersistentVolumeClaim(obj.Name, obj.Namespace)
-			shouldResize, err := k.filterPureVolumesIfEnabled(updatedPVC)
-			if err != nil {
-				return nil, err
-			}
-			if shouldResize {
-				vol, err := k.ResizePVC(ctx, updatedPVC, 1)
+			// For raw block volumes resize is failing hence skipping test for it. defect filed - PWX-32793
```

Comment:
Actually instead of disabling here please disable the test for raw block and make sure to enable again after bug got fixed.